hack/go.mod

```diff
@@ -0,0 +1,413 @@
+module sigs.k8s.io/kustomize/hack
+
+go 1.21
+
+require (
+	github.com/campoy/embedmd v1.0.0
+	github.com/cli/cli v0.6.0
+	github.com/go-bindata/go-bindata/v3 v3.1.3
+	github.com/gohugoio/hugo v0.120.4
+	github.com/golangci/golangci-lint v1.51.2
+	github.com/google/addlicense v1.1.1
+	github.com/goreleaser/goreleaser v0.179.0 // https://github.com/kubernetes-sigs/kustomize/issues/4542
```

Comment:
@karlkfi 
We have already removed `goreleaser` from the release flow.
I think we do not necessarily have this binary.

Reply:
removed